pkg/coord/coord.go

```diff
@@ -892,14 +899,16 @@ func (qc *Coordinator) Split(ctx context.Context, req *kr.SplitKeyRange) error {
 		return spqrerror.Newf(spqrerror.SPQR_KEYRANGE_ERROR, "key range %v already present in qdb", req.Krid)
 	}
 
-	krOldDB, err := qc.qdb.LockKeyRange(ctx, req.SourceID)
+	krOldDB, err := qc.qdb.NoWaitLockKeyRange(ctx, req.SourceID)
 	if err != nil {
 		return err
 	}
 
 	if !req.SplitLeft {
+		// TODO: after convert split command into etcd transaction we no need in embracing "lock" "unlock".
```

Comment:
I don't understand, why did you live these TODOs? I just don't understand what is action item for them? What should we do to fix them i mean